modules/fullscan.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2021 Iliass Alami Qammouri
+#
+# This is free software, licensed under the MIT License.
+# See /LICENSE for more information.
+#
+
+import conf.conf as conf
+
+
+def full_scan():
+    print("===========================================================")
+    print(conf.colored(conf.text2art("All The Scans", "small"), "cyan"))
+    print("===========================================================")
+
+    full_host = input(
+        conf.colored("\nEnter the target URL : ", "green", attrs=["bold"]))
```

Comment:
#Change request parameter
conf.colored("\nEnter the target IP, Hostname or FQDN : ", "green", attrs=["bold"]))